sensor/kubernetes/certrefresh/cert_refresher.go

```diff
@@ -68,10 +64,13 @@ func ensureCertificatesAreFresh(ctx context.Context, requestCertificates request
 	if requestErr != nil {
 		return 0, requestErr
 	}
-	if response.GetError() != nil {
-		return 0, errors.Errorf("central refused to issue certificates: %s", response.GetError().GetMessage())
+	if response.ErrorMessage != nil {
+		return 0, errors.Errorf("central refused to issue certificates: %s", *response.ErrorMessage)
+	}
+	certificates := response.Certificates
```

Comment:
Why not using the getter as it was before?

Reply:
Yes, now response is `certificates.Response`, before it was a protobuf-generated type `central.IssueLocalScannerCertsResponse`